### PR TITLE
Plot 8 columns of debug data

### PR DESCRIPTION
**Pull request by bw1129**
This PR allows Blackbox explorer to properly plot up to 8 columns of debug data (see also PR [#12445](https://github.com/betaflight/betaflight/pull/12445)). The PR makes changes to js/flightlog_fields_presenter.js to accommodate the additional 4 columns. The PR follows the current convention for unused columns, so it is backward compatible with current debug. Thus, adding new columns to existing debug variables is easy and would involve only minor changes to this file for future work. Additionally, this PR updates 'DSHOT_RPM_TELEMETRY' debug to plot RPM data for up to 8 motors (see screenshot below), in accordance with changes made in PR [#12445](https://github.com/betaflight/betaflight/pull/12445). 
Screenshot 2023-02-28 at 7 03 36 PM

### CODE REVIEW

**Comment by nerdCopter** [comment]
Ignore the SonarCloud fail. it reported 85% duplication due to all the new debug inits.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by bw1129** [comment]
@KarateBrot @haslinghuis @blckmn since PR https://github.com/betaflight/betaflight/pull/12445 has been merged, and is dependent upon this one, be great if you're able to review this one

**Comment by KarateBrot** [comment]
We could think about utilizing the 8 debug channels more efficiently by also allowing 2 sets of debugs with 4 channels each in the firmware (instead of hardcoding 8 channel names into the blackbox-log-viewer).

**Comment by bw1129** [comment]
@KarateBrot I think 2 independent debugs sounds fantastic, but it is undoubtedly way more complicated and is beyond the scope of what the current PR is about, but I may not be appreciating what you are thinking in how this could be implemented elegantly. For example, if you wanted two independent 4 column debugs, an advantage would be that you could log GYRO_SCALED at the same time as say DSHOT_RPM_TELEMETRY, which would be fantastic indeed. Thus, I imagine there would be 2 dropdowns in the configurator, and the list of possible debug variables would be duplicated for debug dropdown 1 and debug dropdown 2 (and you could select any combination). But then it is not clear how you would elegantly deal with cases where you want RPM for all 8 motors in an x8 lifter for example (i.e., you'd want both dropdowns to have the same list of debug choices, like DSHOT_RPM_TELEMETRY, but in this case you'd need something like "DSHOT_RPM_TELEMETRY_MOTORS1-4" for dropdown 1, and "DSHOT_RPM_TELEMETRY_MOTORS5-8" for dropdown 2. I am probably not appreciating how you are imagining this, but as I said, the coding will definitely be more complex and really is beyond the scope of the current PR. The simplcity if the current PR opens a ton of possibilities without having to go down that rabbit hole (but please correct me if I am wrong).

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
